app/views/travel_advice/index.html.erb

```diff
@@ -27,13 +27,15 @@
             </div>
             <div class="travel-advice-notice__content">
               <p class="govuk-body">
-                <strong>From 5 November to 2 December 2020, travelling away from home, including internationally, is restricted from <a href="/guidance/new-national-restrictions-from-5-november">England</a> except in limited circumstances such as for work or for education.</strong> Different rules apply in <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/local-lockdown">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>. You must follow all the rules that apply to you.
+                <strong>
+                  Travel in your area, including international travel, may be restricted because of domestic regulations. Different rules apply in <a href="https://www.gov.uk/guidance/local-restriction-tiers-what-you-need-to-know">England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/coronavirus-regulations-guidance">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>. Follow all the rules that apply to you.
```

Comment:
Nitpick: Could use relative links for the links to GOV.UK content.

Reply:
👍